test/slug.test.ts

```diff
@@ -1,4 +1,4 @@
-import { generateRandomSlugs } from "../slug";
+import { generateRandomSlugs } from "../src/slug";
```

Comment:
Keep the tests inside the src/ folder using the same naming convention. It makes it cleaner to configure the project and keeps the test files nearer the tested files. Keep the test and the tested file in the same folder